custom/icds/tests/test_aww_no_submissions.py

```diff
@@ -22,49 +20,92 @@ def setUpClass(cls):
         cls.domain_obj = create_domain(cls.domain)
 
         def make_user(name, location):
-            user = CommCareUser.create(cls.domain, name, 'password')
+            try:
+                user = CommCareUser.create(cls.domain, name, 'password')
+            except Exception:
+                user = CommCareUser.get_by_username(name)
```

Comment:
did you mean to commit this? we could probably make the domain name be more unique if we need to prevent clashes with other tests that aren't cleaning up

Reply:
https://github.com/dimagi/commcare-hq/pull/21581/commits/2dab21e17c547ba79ec5e5ef910afc4c761ea6ef